Tests/SwiftDocCTests/Benchmark/ExternalTopicsHashTests.swift

```diff
@@ -39,7 +39,7 @@ class ExternalTopicsGraphHashTests: XCTestCase {
     
     func testNoMetricAddedIfNoExternalTopicsAreResolved() throws {
         // Load bundle without using external resolvers
-        let (_, context) = try testBundleAndContext(named: "TestBundle")
+        let (_, context) = try testBundleAndContext(named: "DoNotUseInNewTests")
```

Comment:
nit: Any thoughts on something more explicit like `LegacyBundle_DoNotUseInNewTests` to make it clearer that this is a test bundle?

Reply:
I like it.

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1106/commits/86ce80f96332ae40523d60f9482ad7db1834a6b7